src/test/java/Page_Objects/MainMenuPage.java

```diff
@@ -56,19 +60,22 @@ public WebElement getToggleHelp() {
         return driver.findElement(By.xpath("//*[@data-test='toggle-help']"));
     }
 
-    public WebElement getFullscreenElement() {
-        return driver.findElement(By.cssSelector("[data-test='toggle-fullscreen']"));
-    }
-
     public void toggleFullscreen() {
-        wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("[data-test='toggle-fullscreen']")));

```

Comment:
Since this code (`fullScreenElement`, `toggleFullscreen`, `isFullscreenOff` is duplicated in this Page Object and SongListPage, I things the least-worst idea is to create a new Page Object, e.g. `FullscreenPage` and move this code there. This way the implementation won't be duplicated.

This can be done in a separate PR

Reply:
Ok, done. I added FullscreenPage to this code review fixes